skfda/tests/test_classifier_classes.py

```diff
@@ -0,0 +1,76 @@
+"""Tests classes attribute of classifiers."""
+
+import unittest
+
+import numpy as np
+from sklearn.model_selection import train_test_split
+
+from skfda._utils._sklearn_adapter import ClassifierMixin
+from skfda._utils._utils import _classifier_get_classes
+from skfda.datasets import fetch_growth
+from skfda.exploratory.stats.covariance import ParametricGaussianCovariance
+from skfda.misc.covariances import Gaussian
+from skfda.misc.metrics import l2_distance
+from skfda.ml.classification import (
+    DDClassifier,
+    DDGClassifier,
+    DTMClassifier,
+    KNeighborsClassifier,
+    LogisticRegression,
+    MaximumDepthClassifier,
+    NearestCentroid,
+    QuadraticDiscriminantAnalysis,
+    RadiusNeighborsClassifier,
+)
+from skfda.ml.classification._depth_classifiers import _ArgMaxClassifier
+from skfda.representation import FData
+
+from ..typing._numpy import NDArrayAny
+
+
+class TestClassifierClasses(unittest.TestCase):
+    """Test for classifiers classes."""
+
+    def setUp(self) -> None:
+        """Establish train and test data sets."""
+        X, y = fetch_growth(return_X_y=True)
```

Comment:
I would use synthetic data for this test, as the actual functions are not used.
I would also try a case where `y` consists of string labels and another were numbers are used instead.